### PR TITLE
Update and make readme better

### LINKED ISSUE #6: The readme file is a disaster...
Needs to be rewritten with better content and links to the license.md, code of conduct and could use a contributor.md file as well.

### PR DESCRIPTION
**Pull request by willyb321**
Fixes #6

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
```
Comment (designfrontier) [inline CONTRIBUTING.md]: Add a note about asking if you aren't sure what version number to use

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
```
Comment (designfrontier) [inline CONTRIBUTING.md]: ...and others working on the project.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you
```
Comment (designfrontier) [inline CONTRIBUTING.md]: This should say something closer to:
The maintainers will merge the PR once everything is working and ready to go out into the world.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
```
Comment (designfrontier) [inline CONTRIBUTING.md]: remove this line

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+email, or any other method with the owners of this repository before making a change.
+
```
Comment (designfrontier) [inline CONTRIBUTING.md]: and this one :-)

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
+# Contributing
+
```
Comment (designfrontier) [inline CONTRIBUTING.md]: Would be good to have something along the lines of:

A PR is the beginning of a conversation that will hopefully result in a new release of safe-shrinkwrap. The maintainers and other contributors are here to help you get it ready for release. Sometimes the conversation will end with no changes... but most of the time they will result in a new release.

If this is your first time committing to OSS welcome! Don't worry about if you're doing it right, you are. We're excited to help you make it happen!

**Comment by willyb321** [comment]
Alright I think I've done everything you requested.

**Comment by willyb321** [comment]
Thanks!

**Pull request merged**